src/hooks/useVaultInfo.ts

```diff
@@ -0,0 +1,21 @@
+import { parseAbi } from "viem";
+import { useReadContract } from "wagmi";
+import { ArrakisV2HelperABI } from "~/lib/abis/ArrakisHelper";
+import { WewePool } from "./usePool";
+import { ArrakisVaultABI } from "~/lib/abis/ArrakisVault";
+import { ethers } from "ethers";
+
+export function useVaultInfo(selectedPool: WewePool | undefined) {
+
+    const { data, isPending, isError, error } = useReadContract({
+      abi: ArrakisV2HelperABI,
+      functionName: "totalUnderlyingWithFeesAndLeftOver",
+      address: selectedPool?.address as `0x${string}`,
+      args: [], 
```

Comment:
Requesting the wrong address, you should query Arrakis helper address, not the vault address. 
You can find the arrakis contract addresses here: https://github.com/ArrakisFinance/v2-core/blob/main/deploys.md

the code should be like this:

```
functionName: "totalUnderlyingWithFeesAndLeftOver",
address: "0x89E4bE1F999E3a58D16096FBe405Fc2a1d7F07D6", // add it to address list
args: [selectedPool?.address], 
```